src/InteractiveWindow/Editor/InteractiveWindow.cs

```diff
@@ -1917,7 +1917,7 @@ private void ReplaceProjectionSpans(ReadOnlyCollection<SnapshotSpan> oldProjecti
 
         private static ITrackingSpan CreateTrackingSpan(SnapshotSpan snapshotSpan)
         {
-            return new CustomTrackingSpan(snapshotSpan.Snapshot, snapshotSpan.Span, PointTrackingMode.Negative, PointTrackingMode.Positive);
+            return new CustomTrackingSpan(snapshotSpan.Snapshot, snapshotSpan.Span, PointTrackingMode.Negative, PointTrackingMode.Negative);
```

Comment:
Was this the only place where the two tracking modes didn't agree?  If so, can they be combined?